src/crimson/os/seastore/omap_manager.h

```diff
@@ -17,7 +17,9 @@
 //TODO: calculate the max key and value sizes the current layout supports,
 //	and return errors during insert if the max is exceeded.
 #define OMAP_INNER_BLOCK_SIZE 8192
-#define OMAP_LEAF_BLOCK_SIZE 65536
+#define OMAP_LARGE_LEAF_BLOCK_SIZE 65536
+#define OMAP_SMALL_LEAF_BLOCK_SIZE 16384
+#define OMAP_THRESHOLD_SIZE 8192
```

Comment:
Looks this is the threshold of the value size, so the name should be `OMAP_VALUE_SIZE_THRESHOLD`.

An important consideration around the thresholds is that "a b+tree node can only split once upon an insertion", for example:
1. A key-value pair needs to be inserted into a tree node.
2. The tree node cannot hold the new pair so it is splitted.
3. A key-value pair is inserted into a splitted tree node.

I believe it is important that the step 3 (the insertion after split) must be successful, otherwise it means that 1 node is splitted into >2 nodes -- I don't think it can be handled in the current implementation.

This means the threshold is tricky without constraining the key size, and I think inserting up-to 8K values into 16K nodes is apparently possible to violate the above constraint.